examples/deadline/Local-Zone/python/package/lib/compute_tier.py

```diff
@@ -0,0 +1,99 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import dataclass
+from typing import (
+    List,
+    Optional
+)
+
+from aws_cdk.core import (
+    Construct,
+    Stack,
+    StackProps
+)
+from aws_cdk.aws_ec2 import (
+    IMachineImage,
+    InstanceClass,
+    InstanceSize,
+    InstanceType,
+    IVpc,
+    SubnetSelection,
+    SubnetType
+)
+
+from aws_rfdk import (
+    HealthMonitor,
+    SessionManagerHelper
+)
+from aws_rfdk.deadline import (
+    IRenderQueue,
+    WorkerInstanceFleet
+)
+
+
+@dataclass
+class ComputeTierProps(StackProps):
+    """
+    Properties for ComputeTier
+    """
+    # The VPC to deploy resources into.
+    vpc: IVpc
+    # The availability zones the worker instances will be deployed to. This can include your local
+    # zones, but they must belong to the same region as the standard zones used in other stacks in
+    # this application.
+    availability_zones: List[str]
+    # The IRenderQueue that Deadline Workers connect to.
+    render_queue: IRenderQueue
+    # The IMachineImage to use for Workers (needs Deadline Client installed).
+    worker_machine_image: IMachineImage
+    # The name of the EC2 keypair to associate with Worker nodes.
+    key_pair_name: Optional[str]
+
+
+class ComputeTier(Stack):
+    """
+    The computer tier consists of the worker fleets. We'll be deploying the workers into the
+    local zone we're using.
+    """
+    def __init__(self, scope: Construct, stack_id: str, *, props: ComputeTierProps, **kwargs):
+        """
+        Initializes a new instance of ComputeTier
+        :param scope: The Scope of this construct.
+        :param stack_id: The ID of this construct.
+        :param props: The properties of this construct.
+        :param kwargs: Any kwargs that need to be passed on to the parent class.
+        """
+        super().__init__(scope, stack_id, **kwargs)
+
+        # We can put the health monitor and worker fleet in all of the local zones we're using
+        subnets = SubnetSelection(
+            availability_zones=props.availability_zones,
+            subnet_type=SubnetType.PRIVATE,
+            # one_per_az=True,
```

Comment:
Do we want this left out? We have it as `true` in the Typescript version

Reply:
Added it back in.